src/libs/OptionsListUtils.js

```diff
@@ -466,13 +466,39 @@ function getHeaderMessage(hasSelectableOptions, hasUserToInvite, searchValue, ma
     return '';
 }
 
+/**
+ * Returns the currency list for sections display
+ *
+ * @param {Object} currencyListObject
+ * @param {String} searchValue
+ * @param {Object} selectedCurrency
+ * @returns {Array}
+ */
+function getCurrencyListForSections(currencyListObject, searchValue) {
+    const currencyListKeys = _.keys(currencyListObject);
+    const currencyOptions = _.map(currencyListKeys, currencyCode => ({
+        text: `${currencyCode} - ${currencyListObject[currencyCode].symbol}`,
+        searchText: `${currencyCode} ${currencyListObject[currencyCode].symbol}`,
+        currencyCode,
+    }));
```

Comment:
I wonder if it makes sense to cache the currencyOptions in the Modal, so we don't need to rebuild the list on every key input?

Reply:
+1

Reply:
Not sure what you mean by caching the currencyOptions in the Modal? Do you mean memoizing it?

Reply:
I meant that we could hold a reference to `currencyOptions` within the Modal, then passing it into`getCurrencyListForSections` so that each time it is called we don't have to rebuild the list before filtering it.

Reply:
Got it, thank you. I will do that.